core/src/main/java/org/apache/carbondata/core/dictionary/generator/key/DictionaryMessage.java

```diff
@@ -112,20 +110,12 @@ private DictionaryMessageType getKeyType(byte type) {
       case 1:
         return DictionaryMessageType.DICT_GENERATION;
       case 2:
-        return DictionaryMessageType.TABLE_INTIALIZATION;
-      case 3:
         return DictionaryMessageType.SIZE;
-      case 4:
-        return DictionaryMessageType.WRITE_DICTIONARY;
```

Comment:
1. Why WRITE_DICTIONARY is removed...any specific reason?
2. Case 1 and default correspond to same behavior...case 1 statement can be removed or default case should throw exception